### PR TITLE
Register the page pattern category

### PR DESCRIPTION
**Pull request by ibrahim-kardi**
**Description**
Fixed the opened issue #219

**Screenshots**
image

**Testing Instructions**
When creating a new pattern -> in category field type 'pages'

### CODE REVIEW

**Comment by carolinan** [comment]
Hi!
The name of the category needs to be unique, it needs to be prefixed with `twentytwentyfive`.

The registration needs to be hooked to an `init` hook, please see the explanation and code examples in the theme developer handbook:
https://developer.wordpress.org/themes/patterns/registering-patterns/#registering-a-pattern-category

```
add_action( 'init', 'themeslug_register_pattern_categories' );

function themeslug_register_pattern_categories() {
	register_block_pattern_category( 'themeslug/custom', array( 
		'label'       => __( 'Theme Name: Custom', 'themeslug' ),
		'description' => __( 'Custom patterns for Theme Name.', 'themeslug' )
	) );
}
```

**Comment by carolinan** [comment]
Here is another example from Twenty Twenty-Four:

```
/**
 * Register pattern categories.
 */

if ( ! function_exists( 'twentytwentyfour_pattern_categories' ) ) :
	/**
	 * Register pattern categories
	 *
	 * @since Twenty Twenty-Four 1.0
	 * @return void
	 */
	function twentytwentyfour_pattern_categories() {

		register_block_pattern_category(
			'twentytwentyfour_page',
			array(
				'label'       => _x( 'Pages', 'Block pattern category', 'twentytwentyfour' ),
				'description' => __( 'A collection of full page layouts.', 'twentytwentyfour' ),
			)
		);
	}
endif;

add_action( 'init', 'twentytwentyfour_pattern_categories' );
```

**Comment by ibrahim-kardi** [comment]
Thank you @carolinan , I have updated the code

File: functions.php [hunk 1]
```diff
@@ -85,3 +85,28 @@ function twentytwentyfive_block_styles() {
 	);
 }
 add_action( 'init', 'twentytwentyfive_block_styles' );
+
+/**
+ * Register pattern categories.
+ */
+
```
Comment (carolinan) [inline functions.php]:
```suggestion
```
Let's remove this empty line.

**Review by carolinan — APPROVED** [review]
Thank you, I also test ran it and my sample pattern shows up correctly in the category.

**Pull request merged**